src/components/ExploreCarousel/index.js

```diff
@@ -0,0 +1,159 @@
+                  field={previewText.richText}
+                  components={{
+                    paragraph: ({ children }) => <StyledP>{children}</StyledP>,
+                  }}
+                />
+              </Work>
+            </StyledLink>
+          </ItemContainer>
+        );
+      })}
+    </Carousel>
+  );
+
+  const StackedCarousel = () => (
+    <Carousel
+      responsive={responsiveRead}
+      customRightArrow={<CustomRightArrow />}
+      customLeftArrow={<CustomLeftArrow />}
+      infinite={true}
+    >
+      {works.map((work, idx) => {
+        const {
+          work: {
+            document: {
+              uid,
+              data: { title, thumbnail, preview_text: previewText },
+            },
+          },
+        } = works[idx];
+
+        if (idx % 2 === 1) {
+          const {
+            work: {
+              document: {
+                uid: uidTop,
+                data: {
+                  title: titleTop,
+                  thumbnail: thumbnailTop,
+                  preview_text: previewTextTop,
+                },
+              },
+            },
+          } = works[idx - 1];
+          return (
+            <ItemContainer>
+              <StyledLink to={`../works/${uidTop}`}>
+                <Work key={uidTop}>
+                  <FlexRow>
+                    <StyledArticleImage
+                      alt={titleTop.text}
+                      src={thumbnailTop.url}
+                    />
+                    <FlexContainer>
+                      <H3>{titleTop.text}</H3>
+                      <PrismicRichText
+                        field={previewTextTop.richText}
+                        components={{
+                          paragraph: ({ children }) => (
+                            <StyledP>{children}</StyledP>
+                          ),
+                        }}
+                      />
+                    </FlexContainer>
+                  </FlexRow>
+                </Work>
+              </StyledLink>
+              <StyledLink to={`../works/${uid}`}>
+                <Work key={uid}>
+                  <FlexRow>
+                    <StyledArticleImage alt={title.text} src={thumbnail.url} />
+                    <FlexContainer>
+                      <H3>{title.text}</H3>
+                      <PrismicRichText
+                        field={previewText.richText}
+                        components={{
+                          paragraph: ({ children }) => (
+                            <StyledP>{children}</StyledP>
+                          ),
+                        }}
+                      />
+                    </FlexContainer>
+                  </FlexRow>
+                </Work>
+              </StyledLink>
+            </ItemContainer>
+          );
+        }
+      })}
+    </Carousel>
+  );
+
+  return (
+    <FlexContainer>
+      {sectionuid === "read" || sectionuid === "watch" ? (
+        <div>
+          <H2 bold>{title.text}</H2>
```

Comment:
this is so nit-picky but can you style this to be lowercase or change it in prismic

Reply:
I changed it in prismic!